test_cb/test_regression.py

```diff
@@ -44,9 +44,9 @@ def test_regression(default_value, num_tracks, has_constant):
     }
 
     magma_cb = define_cb(**params)
-    res = compile_to_verilog(magma_cb, magma_cb.name, "test_cb/build/")
```

Comment:
res returns if this ran successfully. rather than removing res we should assert that it is True.